MainClassWithTests.java

```diff
@@ -0,0 +1,256 @@
+package com.localpackege;
+
+import com.local.pages.HomePage;
+import com.local.pages.RegisterNewHotelPage;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openqa.selenium.By;
+
+
+
+public class MainClassWithTests extends BasicClass{
+    private HomePage homePage;
+    private RegisterNewHotelPage registerNewHotelPage;
+
+    @BeforeClass
+    public static void beforeRegistrationPageTest() {
+        chrome.navigate().to("http://localhost:8080/article");
+    }
+
+    @Before
+    public void setUpMethods(){
+        homePage = new HomePage(chrome);
+        registerNewHotelPage = new RegisterNewHotelPage(chrome);
+
+    }
+    @Test
+    public void  verifyOpenNewHomePage(){
+        homePage.hoverArticleMenuButton();
+        homePage.hoverNewMenuButton();
+        homePage.clickHomeMenuButton();
+        boolean isDataSectionISDisplayed = chrome.findElement(By.xpath("//span[text()='Data:']")).isDisplayed();
+        boolean isSAveButtonIsDisplayed = chrome.findElement(By.xpath("//span[text()='Save']")).isDisplayed();
+        Assert.assertTrue("Data section should be displayed", isDataSectionISDisplayed);
+        Assert.assertTrue("Save button should be displayed", isSAveButtonIsDisplayed);
+
+    }
+
+         @Test
+    public void verifyThatNameFieldIsEditable(){
+             homePage.hoverArticleMenuButton();
+             homePage.hoverNewMenuButton();
+             homePage.clickHomeMenuButton();
+            boolean isNameFieldISDisplayed = chrome.findElement(By.xpath("//label[text()='Name:']")).isDisplayed();
```

Comment:
Please use page object\page factory and avoid mixing logic of UI interaction and test logic